### DISCUSSION TITLE
Thai string render

### DISCUSSION THREAD

**Discussion by Nalahar** [empira/PDFsharp#145] [Q&A] [answered]
When I use normal examples from github, and instead of hello world type วันนี้เป็นวันที่สวยงาม i get empty squares. Is thai language supported.
1. If it is, what needs to be setup for sample to start rendering properly,
2. If it is not supported, what needs to be done to enable the thai language in rendering.

**Comment by ThomasHoevel** [comment, marked as answer]
You must use fonts that have the characters you need, otherwise you get a placeholder (usually empty squares).

**Reply by Nalahar** [reply]
After checking font lists at https://learn.microsoft.com/en-us/typography/fonts/windows_11_font_list and using Angsana New font the characters appeared.
